content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -0,0 +1,464 @@
+---
+title: Converting an existing Gatsby project to TinaCMS
+last_edited: '2024-10-25T08:29:23.867Z'
+next: content/docs/cli-overview.mdx
+previous: content/docs/contextual-editing/overview.mdx
+---
+
+> ⚠️ Warning: TinaCMS does not officially support Gatsby. We recommend migrating your Gatsby site to a well supported framework such as [Next.JS](https://tina.io/docs/frameworks/next/overview/) instead.
+
+## Introduction
+
+In this tutorial we'll be showing you how you might go about converting an existing gatsby-mdx blog to use TinaCMS. We've created a starter repo for you to follow along with. The repo is a fork of the official [Gatsby md blog starter](https://www.gatsbyjs.com/starters/gatsbyjs/gatsby-starter-blog).
+
+## Limitations
+
+There are a few limitations with the approach outlined in this guide.
+
+- You don't get visual editing
+- You lose out in Gatsby's image optimization
+- Gatsby uses Github Flavoured Markdown, which TinaCMS doesn't fully support
+
+## Getting started
+
+First you'll want to make sure you have the correct version of node installed. We'll be using the node version specified in the repo's `.nvmrc` file, v20.12.0. You can run the code below to make sure you have the correct version installed.
```

Comment:
Don't specify the node version, as if this isn't touched in a while it'll make the docs look old.

Reply:
Done I removed the section about installing node